test/plugin/scenarios/rabbitmq-scenario/src/main/java/test/apache/skywalking/testcase/rabbitmq/Application.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package test.apache.skywalking.testcase.rabbitmq;
```

Comment:
Please rename the package to `org.apache.skywalking.apm.testcase.*`

Reply:
> Please rename the package to `org.apache.skywalking.apm.testcase.*`

I remember the package name in `test/plugin` should not start with `org.apache.skywalking`?

Reply:
Usually, We recommend `org.apache.skywalking.apm.testcase.*`. 
But testcase needs to enhance some class, `test.org.apache.skywalking.apm.testcase` is also good. That likes `spring-scenario`.

Reply:
> I remember the package name in test/plugin should not start with org.apache.skywalking?

These cases are not following the e2e rules. Because we want to make sure explicit enhance. All classes in `org.apache.skywalking.apm.testcase.*` setup by `SkyWalkingAgent#allSkyWalkingAgentExcludeToolkit`